x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterQueryWithPartialResultsIT.java

```diff
@@ -62,12 +64,15 @@ private static class ClusterSetup {
     void populateIndices() throws Exception {
         local.okIds = populateIndex(LOCAL_CLUSTER, "ok-local", local.okShards, between(1, 100));
         populateIndexWithFailingFields(LOCAL_CLUSTER, "fail-local", local.failingShards);
+        createUnavailableIndex(LOCAL_CLUSTER, "unavailable-local");
```

Comment:
Just to be clear, it handles unavailable indices, but non existing ones are not covered for now?
I am trying to understand if this overlaps with https://github.com/elastic/elasticsearch/pull/129641

Reply:
It should handle all the errors from field-caps.

Reply:
Yeah, that is not an error now, okay